data_processors/s3/services.py

```diff
@@ -143,3 +144,34 @@ def tag_s3_object(bucket_name: str, key: str):
     else:
         # sound of silence
         pass
+
+def persist_sequencing_report(bucket: str, key: str, payload: object):
+    # TODO:
+    #  1. Check input sizes and adjust lambda runtimes?
+    query_set = Report.objects.filter(bucket=bucket, key=key)
+    new = not query_set.exists()
+
+    if new:
+        logger.info(f"Creating a new Report key={key}")
+        report = Report(
+            bucket=bucket,
```

Comment:
I reckon this is still WIP? bucket, key kwargs are not belong to Report. Just in case remind here.

Reply:
Very WIP ;) Just wanted a bit of overall read, not details yet ;)